RpcNep5Tracker/RpcNep5Tracker.cs

```diff
@@ -40,7 +41,7 @@ public override void Configure()
             if (_db == null)
             {
                 var dbPath = GetConfiguration().GetSection("DBPath").Value ?? "Nep5BalanceData";
-                _db = DB.Open(dbPath, new Options { CreateIfMissing = true });
+                _db = DB.Open(Path.GetFullPath(dbPath), new Options { CreateIfMissing = true });
```

Comment:
Is this well rebased? i think that this change comes from another PR

Reply:
Yes, I sync it from NEO3's PR. When two nodes installed RpcNep5Tracker, they aren't able to run together on the same PC because they're using the same dbPath (relative path). This PR is able to solve this issue . I've tested it on my local.